unlock-app/src/components/interface/checkout/main/Payment.tsx

```diff
@@ -425,22 +365,28 @@ export function Payment({ checkoutService }: Props) {
                   >
                     <div className="flex justify-between w-full">
                       <h3 className="font-bold">
-                        Pay with {route.symbol} on {route.networkName}
+                        Pay with {route.tokenPayment.symbol} on{' '}
```

Comment:
that throws. an error when the lock is priced in native token. Added a fix

Reply:
ha! Good catch I will look at this gain.